build/Dockerfile.devel

```
@@ -50,8 +50,8 @@ RUN cp -iv /usr/local/bin/clang++ /usr/local/bin/clang++.deref &&\
 	ldconfig &&\
 	rm -rf /mnt/artifacts
 
-LABEL version=0.11.9
-ENV DOCKER_IMAGEVER=0.11.9
+LABEL version=0.11.10
+ENV DOCKER_IMAGEVER=0.11.10
```

Another PR has updated the Docker version. You will need to merge in the new changes and increment the version again.
```suggestion
LABEL version=0.11.11
ENV DOCKER_IMAGEVER=0.11.11
```